### ISSUE TITLE
Block URL Shorteners From Being Allowed

### ISSUE THREAD

**Issue by ThaBiGGDoGG** [Steemhunt/web#182]
Was looking at a hunt. Because I knew it was probably an eCommerce site they linked. https://steemhunt.com/@turboucoin/produits-angcen-2017-nouveau-sunglasses

But the link they gave was a Bit.ly link: http://bit.ly/2LPJMWx?ref=steemhunt That made me think, what if this person was linking it to a website that didn't belong, or to a site with a trojan. There is no reason to use a URL shortener on SteemHunt. You just copy & paste the link on the new post. Using Bit.ly or any of the others would allow people to start to double post more hunts & make more work for mods.

I suggest blocking known URL shorteners to prevent this from happening.

bit.ly
tinyurl.com

I know it is hard to get them all, because there are tons of them, but once a MOD or someone else see's what ones people use, those could be reported & added to the blacklist.

**Comment by superoo7** [comment]
Sample code from @quochuy who developed a chrome extension that expand url
https://github.com/quochuy/steemedphish/commit/4073e5bfeb929b2ecf836e0568807f8b9d2e861e

Use of Link Expansion API
https://tools.steemulant.com/api/unshorten.php?url=http://bit.ly/2LPJMWx?ref=steemhunt

**Comment by sydneyitguy** [comment]
yeah I guess we better use unshort API to get the actual destination.
We're currently having an issue with putting our ref tag "?ref=steemhunt" because some websites just can't handle parameters properly and returns 404 on additional params.
Probably have to check that as well.
